doc/overview.rst

```diff
@@ -773,11 +773,10 @@ Deployment may take about 5 minutes.
        instance_type=instance_type,
    )
 
-Because ``catboost`` and ``lightgbm`` rely on the PyTorch Deep Learning Containers
-image, the corresponding Models and Endpoints display the “pytorch”
-prefix when viewed in the AWS console. To verify that these models
-were created successfully with your desired base model, refer to
-the ``Tags`` section.
+Because the model and script URIs are owned by JumpStart, the endpoint,
```

Comment:
I am not a fan of 'are owned by JumpStart' in public documentation. Besides, isn't it and/or, ie if a customer only uses their own script with a model URIs from JumpStart, would the job use this prefix? How about 'are distributed by SageMaker JumpStart' ?